ras_party/controllers/account_controller.py

```diff
@@ -401,6 +403,50 @@ def verify_token(token, session):
     return {"response": "Ok"}
 
 
+@with_db_session
+def add_respondent_password_token(respondent_id, token, session):
+    """
+    Adds the token to the respondent's password_verification_token column
+
+    :param respondent_id: the respondent's id
+    :param token: The verification token
+    :param session: A db session
+    :return: None on success
+    """
+
+    respondent = query_respondent_by_party_uuid(respondent_id, session)
+    if not respondent:
+        logger.info("Respondent with party id does not exist", respondent_id=respondent_id)
+        raise NotFound("Respondent id does not exist")
+
+    add_respondent_password_verification_token(respondent_id, token, session)
+
+
+@with_db_session
+def delete_respondent_password_token(respondent_id, token, session):
+    """
+    Deletes the token to the respondent's password_verification_token column
+
+    :param respondent_id: the respondent's id
+    :param token: the password verification token
+    :param session: A db session
+    :return: None on success
+    """
+
+    respondent = query_respondent_by_party_uuid(respondent_id, session)
+    if not respondent:
+        logger.info("Respondent with party id does not exist", respondent_id=respondent_id)
+        raise NotFound("Respondent id does not exist")
+    if not respondent.password_verification_token:
+        logger.info("Verification token not found")
+        raise NotFound("Verification token not found")
+    if not token:
+        logger.info("Verification token not received")
+        raise NotFound("Verification token not received")
```

Comment:
This would be a 400 Bad Request

```suggestion
        raise BadRequest("Verification token not received")
```